plugins/modules/copr.py

```diff
@@ -103,6 +103,29 @@
 from ansible.module_utils.basic import AnsibleModule  # pylint: disable=import-error
 from ansible.module_utils.urls import open_url  # pylint: disable=import-error
 
+HAS_RESPAWN_UTIL = False
+if not HAS_DNF_PACKAGES:
+    try:
+        from ansible.module_utils.common import respawn
+    except ImportError:
```

Comment:
This `ImportError `should not be necessary. Respawning is available since ansible-core 2.11, which is the minimum requirement for this module.

Reply:
I thought we may still need to support ansible 2.9 and ansible-base, but it seems that is not the case. My downstream backports to v3 and v4 will still need this, because those versions support older ansible versions.

Reply:
https://github.com/ansible-collections/community.general/pull/6522/commits/934389f9fbafceae5d3f271413b2618a2ab2c263

Reply:
community.general 3 and 4 are EOL, do you imply that they are still actively maintained somewhere downstream?

Reply:
We have v3 packaged in EPEL 8 and v4 packaged in EPEL 9. I'll probably update EPEL 8 to v7 next week and EPEL 9 to whatever the latest version is in six months when RHEL 9.3 is released